portfolio/src/main/java/com/google/sps/data/ContactMe.java

```diff
@@ -0,0 +1,20 @@
+package com.google.sps.data;
+
```

Comment:
Best practice is to leave a comment above the class declaration explaining its use case - for example, "/** Class containing server statistics. */" in ServerStats.java